docs/deploy-development-rp.md

```diff
@@ -17,11 +20,59 @@
 
 1. Install the [Azure
    CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli), if you
-   haven't already.
+   haven't already. You will need `az` version 2.0.72 or greater.
 
 1. Install [OpenVPN](https://openvpn.net/community-downloads), if you haven't
    already.
 
+## Installing the extension
+
+1. Prepare a Python development environment:
+
+   ```
+   make pyenv
+   source pyenv/bin/activate
+   ```
+
+1. Build the development `az aro` extension:
+
+   `make az`
+
+   Note: you may see a message like the following; if so you can safely ignore
+   it:
+
+   ```
+   byte-compiling build/bdist.linux-x86_64/egg/azext_aro/vendored_sdks/azure/mgmt/redhatopenshift/v2019_12_31_preview/models/_models_py3.py to _models_py3.pyc
+     File "build/bdist.linux-x86_64/egg/azext_aro/vendored_sdks/azure/mgmt/redhatopenshift/v2019_12_31_preview/models/_models_py3.py", line 45
+       def __init__(self, *, visibility=None, url: str=None, ip: str=None, **kwargs) -> None:
+                        ^
+    SyntaxError: invalid syntax
+    ```
```

Comment:
Think we can just drop this - Python 3 is a prerequisite now, so if people see this it is in an indicator that they are using 2 when they should use 3.